CITATION.cff

```diff
@@ -20,11 +20,10 @@ authors:
   - given-names: Gregory
     family-names: Way
     orcid: 'https://orcid.org/0000-0002-0503-9348'
-repository-code: 'https://github.com/cytomining/pycytominer-transform'
-url: 'https://cytomining.github.io/pycytominer-transform/'
+repository-code: 'https://github.com/cytomining/CytoTable'
+url: 'https://cytomining.github.io/CytoTable/'
 abstract: >-
-  Transform single cell morphology data for image-based profiling applications and analyses.
-  readouts with Pycytominer.
+  Transform data for processing image-based profiling readouts with Pycytominer and other Cytomining tools.
```

Comment:
```suggestion
  Transform CellProfiler and DeepProfiler data for processing image-based profiling readouts with Pycytominer and other Cytomining tools.
```

Reply:
Thank you for this! Committing into an upcoming batch.